playground/ty/src/Editor/Editor.tsx

```diff
@@ -206,7 +206,8 @@ class PlaygroundServer
   private getVendoredPath(uri: Uri): string {
     // Monaco parses "vendored://stdlib/typing.pyi" as authority="stdlib", path="/typing.pyi"
     // We need to reconstruct the full path
-    return uri.authority ? `${uri.authority}${uri.path}` : uri.path;
+    const fullPath = uri.authority ? `${uri.authority}${uri.path}` : uri.path;
+    return fullPath.startsWith("/") ? fullPath.slice(1) : fullPath;
   }
```

Comment:
I don't think this is the right place. we need to prevent users from creating a file named `vendored://` , similar to what we've done in https://github.com/astral-sh/ruff/pull/20666